src/raft/raft.rs

```diff
@@ -766,13 +816,13 @@ impl<T: Storage> Raft<T> {
                 // also mean that this node has advanced its term number during a network
                 // partition, and it is now unable to either win an election or to rejoin
                 // the majority on the old term. If checkQuorum is false, this will be
-                // handled by incrementing term numbers in response to MsgVote with a higher
-                // term, but if checkQuorum is true we may not advance the term on MsgVote and
-                // must generate other messages to advance the term. The net result of these
+                // handled by incrementing term numbers in response to MsgRequestVote with a higher
+                // term, but if checkQuorum is true we may not advance the term on MsgRequestVote
```

Comment:
Why change these msg types? The original PR doesn't have these changes.
Please take a **close** look.